opencog/nlp/learn/link-pipeline.scm

```diff
@@ -128,12 +128,7 @@
 ; it parsed, and then updates the counts for the observed words and word
 ; pairs.
 (define (observe-text plain-text)
-	(begin
-		(relex-parse plain-text)  ;; send plain-text to server
-		(update-link-counts (get-new-parsed-sentences))
-		(release-new-parsed-sents)
-		(delete-sentences)
-	)
+	(update-link-counts (nlp-parse plain-text))
```

Comment:
Also, be aware that the language learning code is doing very non-standard counting; it has its own independent system based on entropy, which niether probabilistic nor fuzzy tv know about. Also, the merge function for these will break the entropy/mutial information mechanism, and so that cannot be used.